client/web/compose/src/components/PageBlocks/RecordEditor.vue

```diff
@@ -147,6 +148,22 @@ export default {
     horizontal () {
       return this.block.options.horizontalFieldLayoutEnabled
     },
+
+    recordEditorMode () {
+      const overflowMode = 'd-flex gap-2 horizontal-mode'
+      const wrapMode = 'flex-wrap horizontal-mode'
+      let activeMode = ''
```

Comment:
It should be `d-flex flex-column gap-2` by default
Also since now you'll have just one option dictating this, you can have an object (key is option, value is class), so this is more readable.

Reply:
I thought I understood what this meant, but sorry I actually can't seem to figure it out.